### DISCUSSION TITLE
Don't get installer for SQLite3 V3.3.3

### DISCUSSION THREAD

**Discussion by ndcgroup** [pawelsalawa/sqlitestudio#4242] [General]
### Details
Here you describe your problem details, or proposal for enhancement.
After I downloaded  SQLite3 V3.3.3 I never got the installer/wizard to install it

1) typed in "https://sqlitestudio.pl"
2) went to a website for sqlitestudio with a yellow download button and clicked on it
3)double clicked on sqlitestudio-3.3.3.zip
4) Got SQLite Studio Folder on my PC
5) double clicked on SQLitestudio.exe
6) clicked on pop up to "extract all"
7) Put SQLiteStudio.exe shortcut on my desktop
8) Clicked on it but didn't get the installer/wizard (this is the probem)

I have a new Dell Latitude 5510 PC running MS Windows Pro

### Steps to reproduce
This applies only to bug reports. If this is a feature request, delete this section.

### Operating system
This applies only to bug reports. If this is a feature request, delete this section.

### SQLiteStudio version
This applies only to bug reports. If this is a feature request, delete this section.

**Comment by pawelsalawa** [comment]
Version 3.3.x does not come with installer.

After you unzip the package, you just run SQLiteStudio.exe. Wherever you unzip it - this is where you have it installed. You probably want to unzip it to Program Files folder (or move entire unzipped SQLiteStudio folder into Program Files folder). Then create a shortcut on the desktop pointing to Program Files\SQLiteStudio\SQLiteStudio.exe

Good news is that version 3.4.x will bring back the installer.

**Comment by ndcgroup** [comment]
Thank you for your prompt response! I got it working thanks to you!

Nick

**Comment by ygoe** [comment]
I was also searching for the installer. I can install this manually (just replaced the old installed files and kept the start menu shortcut, messing up the software installation registry but I don't care). But my coworker doesn't have that skill.

Looks like the 3.4.0 milestone is really huge and only halfway completed which means it will take forever to get back the installer. We need to upgrade to a newer version of the SQLite engine for new SQL support.

I think I made my own setup for SQLite Studio in the past when there wasn't an official one yet. I might need to update it again to provide the missing setup myself.

PS: This issue might be assigned to the 3.4.0 milestone if that should fix it.

**Reply by pawelsalawa** [reply]
It's true that 3.4.0 still has quite few items on the TODO list.

If you want current (unfinished) 3.4.0 installer, you can grab it from here: https://github.com/pawelsalawa/sqlitestudio/suites/4162723238/artifacts/106944823 - I have just built it.

Also there is a GitHub Sponsors 2nd tier, which grants a sponsor capability of requesting built snapshot - like above - at any moment of development.

**Reply by pawelsalawa** [reply]
(be aware that this is development snapshot and may be less stable then release versions)
